### PR TITLE
New: Post announcing Automattic donation

### PR DESCRIPTION
**Pull request by nzakas**
This the blog post announcement for the Automattic donation.

### CODE REVIEW

File: _posts/2020-12-10-automattic-platinum-sponsor-eslint.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<p class="text-center">
+    <a href="https://automattic.com/" title="Automattic" rel="noopener nofollow" target="_blank"><img class="lazyload" width="400" data-src="/assets/img/logos/automattic.svg" alt="Indeed" src="/assets/img/logos/automattic.svg"></a>
+</p>
+
+We are happy to share that [Automattic](https://automattic.com) has becomes ESLint's first platinum sponsor, donating $2,000 each month for the ongoing maintenance and development of ESLint! Automattic is the company behind <a href="https://wordpress.com">WordPress.com</a>, <a href="https://woocommerce.com">WooCommerce</a>, <a href="https://jetpack.com">Jetpack</a>, <a href="https://simplenote.com/">Simplenote</a>, <a href="https://longreads.com/">Longreads</a>, <a href="https://vaultpress.com/">VaultPress</a>, <a href="https://akismet.com/">Akismet</a>, <a href="https://en.gravatar.com/">Gravatar</a>, <a href="https://crowdsignal.com/">Crowdsignal</a>, <a href="https://cloudup.com/">Cloudup</a>, <a href="https://www.tumblr.com">Tumblr</a>, and more. Further, Automattic believes  in open source and makes a lot of their work [public on GitHub](https://github.com/Automattic) and available via the GPL.
```
Comment (btmills) [inline _posts/2020-12-10-automattic-platinum-sponsor-eslint.md:16]:
```suggestion
We are happy to share that [Automattic](https://automattic.com) has become ESLint's first platinum sponsor, donating $2,000 each month for the ongoing maintenance and development of ESLint! Automattic is the company behind <a href="https://wordpress.com">WordPress.com</a>, <a href="https://woocommerce.com">WooCommerce</a>, <a href="https://jetpack.com">Jetpack</a>, <a href="https://simplenote.com/">Simplenote</a>, <a href="https://longreads.com/">Longreads</a>, <a href="https://vaultpress.com/">VaultPress</a>, <a href="https://akismet.com/">Akismet</a>, <a href="https://en.gravatar.com/">Gravatar</a>, <a href="https://crowdsignal.com/">Crowdsignal</a>, <a href="https://cloudup.com/">Cloudup</a>, <a href="https://www.tumblr.com">Tumblr</a>, and more. Further, Automattic believes  in open source and makes a lot of their work [public on GitHub](https://github.com/Automattic) and available via the GPL.
```

File: _posts/2020-12-10-automattic-platinum-sponsor-eslint.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Here's what Automattic engineers told us about ESLint:
+
+> "ESLint helps us keep our code consistent and clean across hundreds of developers."
+> <br>-- Marcus Kazmierczak, Engineering Team Lead, Automattic
+>
+> "ESLint lets me apply my brain to more important things."
+> <br>-- Darren Ethier, Engineering Team Lead, Automattic
+>
+> "Unlike my variables, ESLint is always used."
```
Comment (btmills) [inline _posts/2020-12-10-automattic-platinum-sponsor-eslint.md:26]: This one is great 😆

**Review by btmills — DISMISSED** [review]
Just one typo, then LGTM!

**Review by btmills — APPROVED** [review]

File: _posts/2020-12-10-automattic-platinum-sponsor-eslint.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+tags:
+  - Open Source
+  - Funding
+  - Open Collective
+  - Sponsors
+  - Backers
+---
+# Automattic becomes ESLint's first platinum sponsor
+
+<p class="text-center">
```
Comment (nzakas) [inline _posts/2020-12-10-automattic-platinum-sponsor-eslint.md:12]:
```suggestion
<p class="text-center" style="margin-top: 100px; margin-bottom: 100px">
```

**Pull request merged**